test/testsyncvirtualfiles.cpp

```diff
@@ -882,27 +906,34 @@ private slots:
 
         QVERIFY(fakeFolder.applyLocalModificationsAndSync());
 
-        QVERIFY(fakeFolder.currentLocalState().find(QStringLiteral("f1") + Theme::instance()->appDotVirtualFileSuffix()));
-        QVERIFY(fakeFolder.currentLocalState().find(QStringLiteral("A/a1") + Theme::instance()->appDotVirtualFileSuffix()));
-        QVERIFY(fakeFolder.currentLocalState().find(QStringLiteral("A/a3") + Theme::instance()->appDotVirtualFileSuffix()));
-        QVERIFY(fakeFolder.currentLocalState().find(QStringLiteral("A/B/b1") + Theme::instance()->appDotVirtualFileSuffix()));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("f1"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/a1"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/a3"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/B/b1"))));
 
         // Make local changes to a3
-        fakeFolder.localModifier().remove(QStringLiteral("A/a3") + Theme::instance()->appDotVirtualFileSuffix());
-        fakeFolder.localModifier().insert(QStringLiteral("A/a3") + Theme::instance()->appDotVirtualFileSuffix(), 100_b);
+        fakeFolder.localModifier().remove(fileName(QStringLiteral("A/a3")));
+        fakeFolder.localModifier().insert(fileName(QStringLiteral("A/a3")), 100_b); // TODO: should this really have the suffix-vfs file suffix??
```

Comment:
@TheOneRing What do you think?

Reply:
Can you check what happened here previously?
With suffix files that should result in an invalid placeholder that was no longer synced.
So after this change the local and remote state should differ, we should have a "A/a3.owncloud" locally but no corresponding file on the remote.